src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/General/CoreType.cs

```diff
@@ -53,6 +53,12 @@ internal enum CoreType
         // For custom attribute processing
         Type,
 
+        // For calling convention processing
+        CallConvCdecl,
+        CallConvFastcall,
+        CallConvStdcall,
+        CallConvThiscall,
```

Comment:
```suggestion
        // For calling convention processing
        CallConvCdecl,
        CallConvStdcall,
        CallConvThiscall,
        CallConvFastcall,
```

I think the order everywhere tends to be Cdecl, Stdcall, Thiscall, and then Fastcall. I'd prefer to keep it consistent unless there is a reason not to.

Reply:
Thanks